rpm/SPECS/docker-ce.spec

```diff
@@ -19,7 +19,11 @@ Requires: docker-ce-cli
 Requires: container-selinux >= 2:2.74
 Requires: libseccomp >= 2.3
 Requires: systemd
+%if 0%{?rhel} >= 8
+Requires: iptables or nftables
```

Comment:
I'm curious; I would've expected the `nftables` package to "provide" `iptables`, is that not the case?

Reply:
I'll file the second PR for the master.

nope - nftables does *not* provide `iptables` 
```shell
$ rpm --provides -q nftables
config(nftables) = 1:0.9.0-8.el8
libnftables.so.0()(64bit)
nftables = 1:0.9.0-8.el8
nftables(x86-64) = 1:0.9.0-8.el8
```